app/src/main/res/layout/cardview_thread_item.xml

```diff
@@ -29,6 +29,8 @@
         android:layout_height="wrap_content"
         android:layout_marginVertical="2dp"
         android:layout_marginStart="@dimen/marginStandardVerySmall"
+        android:nextFocusRight="@+id/newMessageFab"
+        android:nextFocusUp="@id/userAvatar"
```

Comment:
I suspect this nextFocusUp line from spamming the logcat with warnings. The line doesn't seem useful altogether anyway